### DISCUSSION TITLE
ExplorerPatcher blocks whole PC after windows update! How to uninstall?

### DISCUSSION THREAD

**Discussion by Jawomo** [valinet/ExplorerPatcher#1062] [Q&A]
After recent window update i was unable to use the PC anymore. After login screen only the mouse was visible behind a black screen.

I was able to uninstall the update, but now the PC is very unreactive. Task manager doesnt open, Explorer is laggy. Some apps open, but the UI is not visible.

Now i want to uninstall ExplorerPatcher, but when i run the uninstall.exe i see the black screen again and only mouse visible.

How can i get this app uninstalled? It seems to malfunction with latest windows updates installed?

Please Help!

**Comment by Jawomo** [comment]
Managed to get the app uninstalled finally after several reboots und running uninstall.exe.

Now everything seems to be back to normal.

Hopefully the app gets an update to prevent freezing and UI bugs

**Reply by valinet** [reply]
There is indeed a real chance of solving the “problem” since it was through-fully diagnosed and documented, reliably determining what it is and was causing it…

**Comment by Jawomo** [comment]
Seems the app messed up the Explorer. Still having problems.

Any way to reset explorer to default settings? Is the a option to revert all settings in ExplorerPatcher?

**Comment by Aokromes** [comment]
duplicate of https://github.com/valinet/ExplorerPatcher/discussions/1055

**Comment by Jeanyess** [comment]
How did you fix this issue? Just downloaded it and now my whole PC is black and just showing a cursor lol

**Reply by Aokromes** [reply]
press control+alt+del go task manager, run cmd go c:\program files\ExplorerPatcher
then run ep_setup.exe /uninstall till a fix is found.

**Reply by Jawomo** [reply]
@Jeanyess 
Restart pc. When youre on logon screen hold shift button and click restart. You will see a menu on restart then. In the menu theres an revert latest update button.

After this i was able to logon pc again and uninstall the app.

**Reply by Jawomo** [reply]
@Aokromes if you have that black screen you will not be able to run taskmanager or any other app...

**Reply by Aokromes** [reply]
> @Aokromes if you have that black screen you will not be able to run taskmanager or any other app...

ofc you can.

**Reply by Jawomo** [reply]
I was not able to open it. No way.

**Reply by Jesusismygrace** [reply]
Help! I can no longer access my computer after installing this. I can't click on anything to uninstall

**Reply by xxxprod** [reply]
You can open TaskManager with ctrl+alt+delete, find ExplorerPatcher process and kill it, then start explorer.exe from within TaskManager - that should temporarily solve the issue and give access to the desktop where further things like uninstall can be done.

**Reply by dawidmosk** [reply]
Better way is to start CMD as administrator, taskkill /F /IM explorer.exe and run explorer.exe as administrator and just wait few seconds without uninstall.

**Reply by VickySekhon** [reply]
> press control+alt+del go task manager, run cmd go c:\program files\ExplorerPatcher
> 
> then run ep_setup.exe /uninstall till a fix is found.

I can't see the ep_setup.exe. Instead I only see: ![image](https://github.com/user-attachments/assets/e9a337dd-4d65-4fd8-9c32-618b65ed8c82)

How can I fix this!?

I deleted explorer.exe manually through the file explorer.

**Comment by LilyKensa** [comment]
I also encountered the same problem.

My Windows 11 computer was updated last night. As a result, I turned on the computer today, and the login screen is normal, but after logging in, the entire screen is black, there is no desktop, no task bar, only my cursor is visible. What's worse, most of the shortcut keys are not working.

After trying various Windows troubleshooting avenues for a while, I realized it wasn't a Windows issue, and that's when I thought of Explorer Patcher.

The following steps are how I solved this problem:

1. I found that `Ctrl + Alt + Delete` is working, then I opened the task manager.
2. There is a button in the top-right corner to add a task (just like `Windows + R`, but now the hotkey does not respond)
3. Type `cmd` and run as administrator
4. Now, I can use my computer! (without GUI though)
5. Enter at the command line: `cd C:\Program Files\Google\Chrome\Application\`
6. This will take you to the folder where Google Chrome is located, type `dir /b` to see what's there (if you don't have Chrome installed, I guess Edge will work too)
7. Just enter `Chrome.exe` to open Chrome
8. Use Google to find the latest version of Explorer Patcher, download and install it
9. My taskbar, desktop and file explorer are back!

Hope this helps!

**Reply by jcgamage** [reply]
This can be replicated / fixed with a uninstall / reinstall cycle, so it's not catastrophic, but really makes EP unusable / unstable. Hopefully there will be an update for EP soon, or maybe something that detects this kind of post-patch crash  behavior self-disables if explorer is crashing :/

**Reply by bluebynick** [reply]
hi! just buzzing in to provide my slightly modified solution. while i was stuck in this issue, the system wasn’t allowing me to get to chrome via cmd. instead, after your step 2 i opened control panel, then proceeded to find the uninstall section and then uninstall EP. hope this is helpful for someone else reading ur solution ! 🤙🏼

**Reply by VickySekhon** [reply]
> I also encountered the same problem.
> 
> 
> 
> My Windows 11 computer was updated last night. As a result, I turned on the computer today, and the login screen is normal, but after logging in, the entire screen is black, there is no desktop, no task bar, only my cursor is visible. What's worse, most of the shortcut keys are not working.
> 
> 
> 
> After trying various Windows troubleshooting avenues for a while, I realized it wasn't a Windows issue, and that's when I thought of Explorer Patcher.
> 
> 
> 
> The following steps are how I solved this problem:
> 
> 
> 
> 1. I found that `Ctrl + Alt + Delete` is working, then I opened the task manager.
> 
> 2. There is a button in the top-right corner to add a task (just like `Windows + R`, but now the hotkey does not respond)
> 
> 3. Type `cmd` and run as administrator
> 
> 4. Now, I can use my computer! (without GUI though)
> 
> 5. Enter at the command line: `cd C:\Program Files\Google\Chrome\Application\`
> 
> 6. This will take you to the folder where Google Chrome is located, type `dir /b` to see what's there (if you don't have Chrome installed, I guess Edge will work too)
> 
> 7. Just enter `Chrome.exe` to open Chrome
> 
> 8. Use Google to find the latest version of Explorer Patcher, download and install it
> 
> 9. My taskbar, desktop and file explorer are back!
> 
> 
> 
> Hope this helps!

You are such a life saver! God bless you

**Reply by pyrates999** [reply]
Don't run chrome with elevated privileges.  Add the necessary exclusions to windows defender that are listed in the release notes here: https://github.com/valinet/ExplorerPatcher/releases

**Comment by dnichyccny** [comment]
Bump this is still an issue.
I installed this on a few client machines and it happens often.
Preview updates turned off via group p;policy, still happens so it may not by a symbols thing or the monthly CU still has new symbols?

**Comment by moedwyn** [comment]
Ctrl+alt+del=>task manager=>run notepad=>open*showalltype=>rename ep_setup file into ep_uninstall=>run=>solved
*wait untill new update of working ep related to current windows update version

**Comment by pyrates999** [comment]
I think this can be closed now as several answers have been given.

**Reply by dnichyccny** [reply]
Solutions for when it happens but not from how to stop it from happening.

Is it possible to code something into this app that says IF preview build then dont load or throw error message or force old symbols?

**Reply by pyrates999** [reply]
explorer patcher doesn't know you've installed a preview update.  All it knows is that it can't download symbols.

**Reply by dnichyccny** [reply]
Im just sayin for a future update, would be good if it can avoid the issue. Just because of this symbols thing.

**Comment by LarsErikOsterud** [comment]
I have a similar problem. But can't delete the files. They are "locked" (in use).
Neither uninstall or new install works (only says files are locked). How do I "unlock" them?

**Reply by dnichyccny** [reply]
rough, maybe boot windows safe mode?

**Comment by ArtyzAudio** [comment]
having this problem again! hope for new version asap

**Reply by pyrates999** [reply]
Please post windows version and build.
Please post explorer patcher version.

**Comment by dawidmosk** [comment]
Best solution is having ep_setup.exe on disk and Ctrl + Alt  + Del, run -> cmd and ep_setup.exe /uninstall
Should be resolved in some way - like crashing window or something.

**Comment by gorillale** [comment]
Same BlackScreen Today! after windows update (27.9.2023) 
I blocked winupdate 5 months after restore whole pc.
Crap....

**Reply by pyrates999** [reply]
That is a preview update: https://support.microsoft.com/en-us/topic/september-26-2023-kb5030310-os-build-22621-2361-preview-363ac1ae-6ea8-41b3-b3cc-22a2a5682faf

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Comment by gorillale** [comment]
i tryed with taskmanager but no explorerpatcher found...

![update](https://github.com/valinet/ExplorerPatcher/assets/87060392/edbceb8a-268f-4523-8a5f-27ccbb6d7715)

**Reply by dnichyccny** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/66123064/154cba6a-5f7f-4c2e-920c-f6f9f6074da5)
You can run control panel from task manager>program's> uninstall expatecher, reinstall later

**Reply by pyrates999** [reply]
That would be a preview update.  And that button says download and install I believe.  You must ignore updates that present themselves this way.

to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

**Comment by gorillale** [comment]
ok thank you
actual i have updated explorer patcher (i think my version was old).
Now i update again, same update and it works.
Thank you.
Regards

**Comment by Sekai-No-Saiba** [comment]
I was able to fix the issue with the new fix release without much trouble ... but now on my other PC I am not able to go through the following error 

![ep_error](https://github.com/valinet/ExplorerPatcher/assets/133463917/7141dd37-e3e1-4521-b975-d5ff6ad2d5c3)

**Reply by pyrates999** [reply]
Try rebooting.

**Reply by Sekai-No-Saiba** [reply]
I can't do anything to the EP ... even uninstall it ... I have done every basic recommended steps ...   

My current state is to summon Taskmanager to relaunch explorer.exe till EP allows me to "fix the issue" 
so there for , the error of access memory seem to prevent me to either update/uninstall or delete/uninstall 

@valinet      // please help

**Reply by pyrates999** [reply]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
